app/channels/editor_channel.rb

```diff
@@ -12,18 +12,17 @@ class EditorChannel < ApplicationCable::Channel
 
   attr_accessor :resource
 
-  AUTOSAVE_EVENT = 'auto-save'.freeze
-
   def subscribed
     reject and return unless find_resource
 
     stream_for [current_user, current_project, resource]
   end
 
   def save(params)
-    resource.paper_trail_event = AUTOSAVE_EVENT
+    resource.paper_trail_event = RevisionTracking::REVISABLE_EVENTS[:autosave]
 
     if resource.update_attributes resource_params(params)
+      RevisionCollapser.collapse(resource, RevisionTracking::REVISABLE_EVENTS[:autosave])
```

Comment:
I get this is a Channel, but in the interest of having the code in one place, wouldn't we be better off including the `RevisionCollapsing` concern here and calling `#collapse_revisions`?

It does add the background Job, and Channel is already a background channel, I know.

If you feel strongly about that, then maybe referencing this file in the RevisionCollapser form as a "source" or "caller" section, and hoping that'll be enough in future refactors for us to look here.